### DISCUSSION TITLE
How to fix the navbar on top?

### DISCUSSION THREAD

**Discussion by rishavs** [saadeghi/daisyui#2277] [Q&A] [answered]
Hi

I am using the daisy navbar alongwith drawers. If I apply the `fixed` tailwind directive to my navbar, it corrupts the view and the drawer renders on top of my navbar. 
How should I go about making my page header/nav fixed when I am also using drawers?

**Comment by saadeghi** [comment]
What do you mean "it corrupts the view"?  

Can you provide a code example on [Tailwind Play](https://daisyui.com/tailwindplay/)?

**Comment by rishavs** [comment]
@saadeghi , here is the sample code https://play.tailwindcss.com/kayyB7MXD5?size=1048x720

If you remove the fixex attribute from the first line, my site looks fine. But if I try to fix the navbar, all items start overlapping with each other in the y axis and the side panel renders on top of the navbar.

**Reply by rikgirbes** [reply, marked as answer]
Yeah thats because of fixed. `fixed` basically removes an element from all normal flow of the page.

You'd have to give it proper top/right/left/bottom values.

Solution: you could just use sticky 😊 

```html
<header class="navbar sticky top-0 z-50 bg-neutral">
```

It'll just stay in the normal flow of the page, top-0 makes sure it knows where to "stick", and z-index is needed otherwise it would land "underneath" your sidebar.

![Screenshot 2023-10-27 at 09 53 12](https://github.com/saadeghi/daisyui/assets/12097579/c5e995f2-4f84-40ea-a7b6-6fb7343bcdd9)

https://play.tailwindcss.com/3aJfqd22IY?size=1048x529
